compiler/src/main/java/io/neow3j/compiler/sourcelookup/ISourceContainer.java

```diff
@@ -0,0 +1,18 @@
+package io.neow3j.compiler.sourcelookup;
+
+import java.io.File;
+import java.util.List;
+
+
+public interface ISourceContainer {
+
+    /**
+     * Searches this container for the source file(s) with the given name. Returns multiple files
+     * if multiple items with the same name are found.
+     *
+     * @param name The name of the file.
+     * @return a list of sources files found matching the given name.
+     */
+    File findSourceFile(String name);
```

Comment:
In the javadoc above it mentions multiple files, but here it's just `FIle` and not a `List<File>`. Should we change to be a bit more flexible and implement as a `List`?